stage0_bin/layout.ld

```diff
@@ -300,7 +314,7 @@ SECTIONS {
 
         /* SEV-ES reset block: uint32 addr, uint16 size, 16-byte GUID */
         HIDDEN(sev_es_reset_block_start = .);
-        LONG(0xDEADBEEF) /* Placeholder values */
+        LONG(ap_start) /* Placeholder values */
```

Comment:
The comment seems out of date.

Reply:
Good point, it's indeed no longer a placeholder.